resources/zib/zib-HeadCircumference.xml

```diff
@@ -0,0 +1,130 @@
+      <alias value="Schedelomvang" />
+      <mapping>
+        <identity value="zib-headcircumference-v1.3-2020EN" />
+        <map value="NL-CM:12.14.1" />
+        <comment value="HeadCircumference" />
+      </mapping>
+    </element>
+    <element id="Observation.code">
+      <path value="Observation.code" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://loinc.org" />
+          <code value="9843-4" />
+        </coding>
+      </patternCodeableConcept>
+    </element>
+    <element id="Observation.effective[x]">
+      <path value="Observation.effective[x]" />
+      <slicing>
+        <discriminator>
+          <type value="type" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Observation.effective[x]:effectiveDateTime">
+      <path value="Observation.effective[x]" />
+      <sliceName value="effectiveDateTime" />
+      <short value="HeadCircumferenceDate" />
+      <definition value="The date on which the head circumference was measured." />
+      <alias value="SchedelomvangDatumTijd" />
+      <type>
+        <code value="dateTime" />
+      </type>
+      <mapping>
+        <identity value="zib-headcircumference-v1.3-2020EN" />
+        <map value="NL-CM:12.14.2" />
+        <comment value="HeadCircumferenceDate" />
+      </mapping>
+    </element>
+    <element id="Observation.value[x]">
+      <path value="Observation.value[x]" />
+      <slicing>
+        <discriminator>
+          <type value="type" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Observation.value[x]:valueQuantity">
+      <path value="Observation.value[x]" />
+      <sliceName value="valueQuantity" />
+      <short value="HeadCircumferenceMeasurement" />
+      <definition value="The value of the head circumference as measured in centimeters." />
+      <comment value="The zib states that both cm and mm can be used as unit. However. the HL7 Vital Signs profile restricts the metric units to just cm. A value in mm SHALL therefore be communicated as a fractional cm." />

```

Comment:
This is not completly correct: the Vital Signs profile allows _cm_ and _in_i_ : https://hl7.org/FHIR/headcircum.html

Also a small typo after however: should be a ',' instead of a dot.  Agree with the fractional CM though.

Reply:
_in_i_ is not metric. I'll fix the dot.